src/box/flightrec.h

```diff
@@ -21,23 +21,42 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
+/** Flight recorder config options. */
+struct flight_recorder_cfg {
+	/**
+	 * Directory to store flight_records.ttfr file.
+	 * Should be the same as memtx_dir.
+	 */
+	const char *dir;
+	/** Total size of stored logs. */
+	int64_t logs_size;
+	/** Max size of one log message. */
+	int64_t log_max_msg_size;
+	/** Flight recorder log level; may be different from say log level. */
+	int logs_log_level;
+	/**
+	 * Time interval (in seconds) between metrics
+	 * dump to flight recorder.
+	 */
+	double metrics_interval;
+	/**
+	 * Period (in seconds) of metrics storage, i.e. how long
+	 * metrics are stored before being overwritten.
+	 */
+	int64_t metrics_period;
+	/** Total size of stored requests and responses. */
+	int64_t requests_size;
+	/** Requests_max_req_size - Max size per request. */
+	int64_t requests_max_req_size;
+	/** requests_max_res_size - Max size per response. */
```

Comment:
Nit: Please remove `Requests_max_req_size -` and `requests_max_res_size -` from the last two comments.